ahoy.js

```diff
@@ -359,13 +360,25 @@
   };
 
   ahoy.trackAll = function() {
+    ahoy.start();
     ahoy.trackView();
     ahoy.trackClicks();
     ahoy.trackSubmits();
     ahoy.trackChanges();
   };
 
-  $(createVisit);
+  ahoy.start = function (options) {
+    ahoy.configure(options || {});
+    createVisit();
+
+    for (var i = 0; i < eventQueue.length; i++) {
+      trackEvent(eventQueue[i]);
+    }
+
+    ahoy.start = function () {};
+  };
+
+  if (config.startOnReady) { $(function () { ahoy.start(); }); }
```

Comment:
This will actually cause problems, since start accepts an options hash; $(start) will pass the event as the first parameter.